diff -ruN openvas-scanner-5.1.2/CMakeLists.txt a/CMakeLists.txt
--- openvas-scanner-5.1.2/CMakeLists.txt	2018-03-07 06:54:14.000000000 -0500
+++ a/CMakeLists.txt	2018-05-14 15:57:44.452691266 -0400
@@ -226,7 +226,7 @@
 set (LINKER_HARDENING_FLAGS     "-Wl,-z,relro -Wl,-z,now")
 set (GPGME_C_FLAGS              "-D_FILE_OFFSET_BITS=64 -DLARGEFILE_SOURCE=1")
 
-set (CMAKE_C_FLAGS_DEBUG        "${CMAKE_C_FLAGS_DEBUG} ${GPGME_C_FLAGS} -Werror -Wextra")
+set (CMAKE_C_FLAGS_DEBUG        "${CMAKE_C_FLAGS_DEBUG} ${GPGME_C_FLAGS} -Wextra")
 set (CMAKE_C_FLAGS              "${CMAKE_C_FLAGS} ${HARDENING_FLAGS} ${GPGME_C_FLAGS} -Wall -D_BSD_SOURCE -D_ISOC99_SOURCE -D_SVID_SOURCE -D_DEFAULT_SOURCE")
 
 add_subdirectory (src)
