diff -ruN openvas-scanner-5.0.5/CMakeLists.txt openvas-scanner-5.0.5-new/CMakeLists.txt
--- openvas-scanner-5.0.5/CMakeLists.txt	2015-08-03 06:14:45.000000000 -0400
+++ openvas-scanner-5.0.5-new/CMakeLists.txt	2016-01-13 10:58:42.146637896 -0500
@@ -227,7 +227,7 @@
 
 set (HARDENING_FLAGS            "-Wformat -Wformat-security -O2 -D_FORTIFY_SOURCE=2 -fstack-protector -Wl,-z,relro -Wl,-z,now")
 
-set (CMAKE_C_FLAGS_DEBUG        "${CMAKE_C_FLAGS_DEBUG} -Werror")
+set (CMAKE_C_FLAGS_DEBUG        "${CMAKE_C_FLAGS_DEBUG} ")
 set (CMAKE_C_FLAGS              "${CMAKE_C_FLAGS} ${HARDENING_FLAGS} -Wall -D_BSD_SOURCE -D_ISOC99_SOURCE -D_SVID_SOURCE -D_DEFAULT_SOURCE")
 
 add_subdirectory (src)
